lib/library/books/book_loan.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Library.Books.BookLoan do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Library.Books.BookLoan
+
+
+  schema "book_loans" do
+    field :queue, {:array, :integer}
+    field :book_id, :id
+    field :user_id, :id
```

Comment:
In your migration `book_id` and `user_id` are referencing other tables. Should this be 'belongs_to'  instead? Same in the `Requests` schema.